datadog_checks_dev/datadog_checks/dev/tooling/commands/__init__.py

```diff
@@ -22,5 +21,4 @@
     metadata,
     release,
     test,
-    changed,
```

Comment:
Are you removing this command ?

Reply:
Oh that's what it does ? I think the docstring could be clearer then. I thought `--list` was just listing the available tox envs for a check

Reply:
Yes that is what both do. Though `ddev test --list` will list changed checks _and_ their envs,

Reply:
Why remove it when they do different things?

Reply:
I'm confused here... they both show the same info but new one has a bit more

Reply:
Also we should try always to not clutter the root command namespace